src/Controls/src/Core/Platform/Windows/CollectionView/ItemContentControl.cs

```diff
@@ -190,10 +190,36 @@ internal void Realize()
 				_visualElement.BindingContext = dataContext;
 			}
 
-			Content = _renderer.ToPlatform();
+			if (_renderer.VirtualView is ICrossPlatformLayout)
+			{
+				Content = _renderer.ToPlatform();
```

Comment:
Fundamentally I think we need the `base.measure` and `base.arrange` of the content control to just be able to flow naturally. There's a lot of behavior inside the base class of `ContentControl` that has to execute so we just need the content to be set to a control type that's able proxy the measure/override up to `IView`

If you look at the measureoverride/measurearrange now we can basically just remove all the code. 

Later down the road once we go to ItemRepeater I wonder if ContentControl is the right base class to use here.